From: Oleksij Rempel <o.rempel@pengutronix.de>
Date: Tue, 9 Jan 2024 16:56:55 +0100
Subject: [PATCH] arm64: dts: freescale: remove pstore

It seems to have some stability issue. Remove it for now.

Signed-off-by: Oleksij Rempel <o.rempel@pengutronix.de>
---
 arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi b/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
index 929cf6f1035f..ee099a3a832c 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mp-skov-reva.dtsi
@@ -177,21 +177,6 @@ reg_vsd_3v3: regulator-vsd-3v3 {
 		gpio = <&gpio2 19 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
 	};
-
-	reserved-memory {
-		#address-cells = <2>;
-		#size-cells = <2>;
-		ranges;
-
-		ramoops: ramoops@bfdd0000 {
-			compatible = "ramoops";
-			reg = <0x00 0xbfdd0000 0x00 0x0100000>;
-			record-size = <0x8000>;
-			console-size = <0x8000>;
-			ftrace-size = <0x00>;
-			pmsg-size = <0x8000>;
-		};
-	};
 };
 
 &A53_0 {
